docs/tutorials/google/xeb_calibration_example.ipynb

```diff
@@ -0,0 +1,674 @@
+{
```

Comment:
<p>this doesn't work as written without a processor_id. Why not set up sampler to optionally be a coherent-error sampler like in the floquet notebook?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/4258/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p>Because the first step is looking at calibration to pick qubits - which can't be done with a coherent-error sampler AFAIK - this notebook really requires a processor_id. So the strategy was to just save all outputs.</p>